witness/README.md

```diff
@@ -76,7 +76,11 @@ defined above.  Currently it contains implementations in the following languages
 
 ## Feeders
 
-* [Go SumDB](https://github.com/google/trillian-examples/tree/master/sumdbaudit/witness) :
-    a custom feeder for sum.golang.org that can be easily deployed with the generic golang witness implementation
-* [Serverless](https://github.com/google/trillian-examples/tree/master/serverless/cmd/feeder) :
-    a feeder command that acquires checkpoints from [serverless logs](https://github.com/google/trillian-examples/tree/master/serverless).
+Specific feeders are listed below, however potential witness operators are advised to
+simply deploy the [Omniwitness](golang/omniwitness) unless there is a compelling reason
```

Comment:
Is it worth adding a parenthetical to explain that the OmniW is bundled with feeders and configs for all known logs?

Reply:
I added a bit more here.